src/Column/ColumnDefinitionParser.php

```diff
@@ -44,4 +46,40 @@ protected function extraInfo(string $extra): array
 
         return $info;
     }
+
+    protected function parseTypeParams(string $type, string $params): array
+    {
+        return match ($type) {
+            'bit',
+            'bigint',
+            'binary',
+            'char',
+            'decimal',
+            'double',
+            'float',
+            'int',
+            'integer',
+            'mediumint',
+            'numeric',
+            'real',
+            'smallint',
+            'string',
+            'tinyint',
+            'varbinary',
+            'varchar',
+            'year' => $this->parseSizeInfo($params),
+            'enum' => $this->parseEnumValues($params),
+            default => [],
+        };
```

Comment:
```suggestion
        return match ($type) {
            'enum' => $this->parseEnumValues($params),
            default => $this->parseSizeInfo($params),
        };
```
It seems a mistake to limit the list of types for parsing.
We use column definition parser not only for native db types but for abstract types also.

Reply:
What case for abstract types?